### PR TITLE
Add warning logging level

### PR DESCRIPTION
**Pull request by ashi009**
Add warning level, to distinguish chatty info and actual errors.

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -75,9 +78,15 @@ func Init(name string, verbose, systemLog bool, logFile io.Writer) *Logger {
 	}
 
 	var l Logger
-	l.infoLog = log.New(io.MultiWriter(iLogs...), "INFO: ", flags)
-	l.errorLog = log.New(io.MultiWriter(eLogs...), "ERROR: ", flags)
-	l.fatalLog = log.New(io.MultiWriter(eLogs...), "FATAL: ", flags)
+	l.infoLog = log.New(io.MultiWriter(iLogs...), tagInfo, flags)
+	l.infoLog = log.New(io.MultiWriter(iLogs...), tagWarning, flags)
```
Comment (adjackura) [inline logger.go]: l.warningLog

File: logger.go [hunk 2]
```diff
@@ -93,16 +102,26 @@ type severity int
 
 const (
 	sInfo = iota
+	sWarning
 	sError
 	sFatal
 )
 
+const (
```
Comment (adjackura) [inline logger.go]: For readability move these above the functions where their used (above initialize)

File: logger.go (hunk 1 above)
Comment (adjackura) [reply logger.go]: Also info only logs to stdout if verbose is set, does this make sense for warning too?

**Review by adjackura — CHANGES_REQUESTED** [review]

File: logger.go (hunk 1 above)
Comment (ashi009) [reply logger.go]: Good catch.  Done.

By default glog logs `ERROR` and above to `stderr`, and `-alsologtostderr` to drop the threshold to `INFO`, so they are consistent.  However as this package doesn't really have a verbosity control, so there is no good way to align them.

File: logger.go (hunk 2 above)
Comment (ashi009) [reply logger.go]: Done.

**Comment by ashi009** [comment]
PTAL

**Comment by adjackura** [comment]
Resolve the conflicts and I'll merge

**Comment by ashi009** [comment]
PTAL

**Review by adjackura — APPROVED** [review]

**Pull request merged**